spec/money_tracker_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe MoneyTracker do
+
+  let(:money_tracker) { MoneyTracker.new }
+
+  it { is_expected.to respond_to(:credit).with(1).arguments }
+  it { is_expected.to respond_to(:debit).with(1).arguments }
+  it { is_expected.to respond_to(:save_transaction).with(3).arguments }
+
+  it "initialises with a balance of 0" do
+    expect(money_tracker.balance).to eq 0
+  end
+
+  describe "#credit" do
```

Comment:
Really easy to read describe block.  Love that the setup is done in the before block.